src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/ProcessRdfForm.java

```diff
@@ -117,40 +109,28 @@ public AdditionsAndRetractions  process(
         return changes;
     }
     
-    public AdditionsAndRetractions  process(
-            EditConfigurationVTwo configuration,
-            MultiValueEditSubmission submission) 
-    throws Exception{  
-        log.debug("configuration:\n" + configuration.toString());
-        log.debug("submission:\n" + submission.toString());
-        
-        //System.out.println("Configurations:");
-        //System.out.println(configuration.toString());
-        //System.out.println("Submissions:");
-        //System.out.println(submission.toString());
-
-        applyEditSubmissionPreprocessors( configuration, submission );
-        
-        AdditionsAndRetractions changes;
-        if( configuration.isUpdate() ){
-            //System.out.println("Edit exsiting ones");
-            changes = editExistingStatements(configuration, submission);
-            //System.out.println(changes.toString());
-        } else {
-            //System.out.println("Create new ones");
-            changes = createNewStatements(configuration, submission );
-            //System.out.println(changes.toString());
-
-		    /* DCO-ID generation, by Han Wang (wangh17@rpi.edu) */
-            generateDcoIdN3(changes, submission);
```

Comment:
Is this code being called from someplace else? Generating our DCOId is necessary.